package.json

```diff
@@ -13,16 +13,45 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onCommand:extension.helloWorld"
+		"onView:tmcView"
```

Comment:
Something to keep in mind: commands registered under this event become available everywhere but only after the specified view is visited at (see `helloWorld`).